TrainingLoop/Callbacks/CSVLogger.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import ModelSupport
+
+/// A handler for logging training and validation statistics to a CSV file.
+public class CSVLogger {
+  /// The path of the file that statistics are logged to.
+  public var path: String
+
+  // The boolean variable indicating if header of the CSV file has been written or not.
```

Comment:
```suggestion
  // True iff the header of the CSV file has been written.
```

Everyone can see it's a `Bool`.  The phrasing “indicating if … or not” is not clear about the polarity of the `Bool`.

Reply:
Got it!